### PR TITLE
Support for different package names in build*.sh

### PR DESCRIPTION
**Pull request by pjh5**
I will run a job to test that this works before removing wip

### CODE REVIEW

**Comment by pjh5** [comment]
This works

File: manywheel/build_cpu.sh [hunk 1]
```diff
@@ -208,10 +208,20 @@ cp /$WHEELHOUSE_DIR/torch*.whl /remote/$WHEELHOUSE_DIR/
...
+echo "Expecting the built wheels to be packages for '$package_name'"
+
+
+# Test that all the wheels work
 export OMP_NUM_THREADS=4 # on NUMA machines this takes too long
 pushd $PYTORCH_DIR/test
 for PYDIR in "${DESIRED_PYTHON[@]}"; do
-    "${PYDIR}/bin/pip" uninstall -y torch
-    "${PYDIR}/bin/pip" install torch --no-index -f /$WHEELHOUSE_DIR
+    "${PYDIR}/bin/pip" uninstall -y "$package_name"
```
Comment (soumith) [inline manywheel/build_cpu.sh]: this is super weird that this works (or is it not uninstalling maybe?). I think to allow a configurable package_name, you need to make changes to pytorch/setup.py, around here: https://github.com/pytorch/pytorch/blob/master/setup.py#L1026

Comment (pjh5) [reply manywheel/build_cpu.sh]: I have those corresponding changes here https://github.com/pytorch/pytorch/pull/9544 . I'm using that PR to test changes in the pytorch repo

Comment (pjh5) [reply manywheel/build_cpu.sh]: The environment variable that this expects PACKAGE_NAME is also written into the CI jenkins code

**Pull request merged**